src/Collection/LeeanMenu.php

```diff
@@ -0,0 +1,131 @@
+<?php namespace Leean\Widgets\Collection;
+
+use Leean\Widgets\Models\AbstractWidget;
+
+/**
+ * Class LeeanMenu.
+ *
+ * @package Leean\Widgets\Collection
+ */
+class LeeanMenu extends AbstractWidget
+{
+	/**
+	 * Constructor.
+	 */
+	public function __construct() {
+		parent::__construct( 'Leean Menu', 'Display a menu' );
+	}
+
+	/**
+	 * Add the menu select.
+	 *
+	 * @param array $instance
+	 */
+	public function form( $instance ) {
+		parent::form( $instance );
+
+		$menus = get_terms( 'nav_menu', array( 'hide_empty' => true ) );
+
+		$selected_menu = isset( $instance['menu'] ) ? $instance['menu'] : '';
+
+		$menu_field_name = $this->get_menu_field_id();
+
+		?>
+
+		<p>
+
+			<label for="<?php echo esc_attr( $menu_field_name ) ?>">Menu:</label>
+
+			<select id="<?php echo esc_attr( $menu_field_name ) ?>"
+					name="<?php echo esc_attr( $menu_field_name ) ?>">
+
+				<?php foreach ( $menus as $menu ) : ?>
+
+					<option value="<?php echo esc_attr( $menu->slug ) ?>"
+						<?php selected( $selected_menu, $menu->slug ); ?> >
+						<?php echo esc_html( $menu->name ) ?>
+					</option>
+
+				<?php endforeach ?>
+
+			</select>
+
+		</p>
+
+		<?php
+	}
+
+	/**
+	 * Save the menu slug
+	 *
+	 * @param array $new_instance
+	 * @param array $old_instance
+	 * @return array
+	 */
+	public function update( $new_instance, $old_instance ) {
+		$menus = get_terms( 'nav_menu', array( 'hide_empty' => true ) );
+
+		$selected_menu = sanitize_text_field( wp_unslash( $_REQUEST[ $this->get_menu_field_id() ] ) );
```

Comment:
We need to check if the index key exists in $_REQUEST before using it
Thats why CI complains